pkg/k8shandler/visualization.go

```diff
@@ -58,7 +58,7 @@ func CreateOrUpdateVisualization(cluster *logging.ClusterLogging) (err error) {
 		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 			if exists, cluster := utils.DoesClusterLoggingExist(cluster); exists {
 				if !reflect.DeepEqual(kibanaStatus, cluster.Status.Visualization.KibanaStatus) {
-					logrus.Infof("Updating status of Kibana")
+					logrus.Info("Updating status of Kibana")
```

Comment:
I think it is useful to print this, but not more than once if looping in the RetryOnConflict